addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -453,6 +453,15 @@ ListRenderer.include({
             delete this.allFieldWidgets[recordID];
         }
     },
+    /**
+     * Check whether list view is ediable listview
+     * consider editable if editable attribute given and edit=true
+     *
+     * @private
+     */
+    _isEditable: function () {
+        return this.editable && (!this.arch.attrs.edit || !!JSON.parse(this.arch.attrs.edit));
```

Comment:
this should be directly encoded in the 'editable' param, so you should move this logic to list_view.js (and use this.controller.Params.activeActions.edit)